crates/db_schema/src/utils.rs

```diff
@@ -557,6 +555,10 @@ pub fn now() -> AsExprOf<diesel::dsl::now, diesel::sql_types::Timestamptz> {
   diesel::dsl::now.into_sql::<Timestamptz>()
 }
 
+pub fn seconds_to_pg_interval(seconds: i32) -> PgInterval {
+  PgInterval::from_microseconds((seconds * 1_000_000).into())
```

Comment:
to prevent integer overflow

```suggestion
  PgInterval::from_microseconds(i64::from(seconds) * 1_000_000)
```